.github/scripts/unittest-linux/install.sh

```diff
@@ -98,14 +98,9 @@ if [[ "$(python --version)" = *3.9* || "$(python --version)" = *3.10* ]]; then
 fi
 (
     set -x
-    conda install -y -c conda-forge ${NUMBA_DEV_CHANNEL} libvorbis parameterized 'requests>=2.20'
-    pip install SoundFile coverage pytest pytest-cov scipy expecttest unidecode inflect Pillow sentencepiece pytorch-lightning 'protobuf<4.21.0' demucs tinytag pyroomacoustics flashlight-text git+https://github.com/kpu/kenlm
+    conda install -y -c conda-forge ${NUMBA_DEV_CHANNEL} parameterized 'requests>=2.20'
+    pip install coverage pytest pytest-cov scipy expecttest Pillow
```

Comment:
Approving to unblock, feel free to merge when green after skipping the currently failing tests.

We shouldn't need Pillow, and we should remove the `pip install` line below as well with the TODO